llvm/tools/llvm-mc/llvm-mc.cpp

```diff
@@ -240,6 +242,24 @@ static cl::opt<ActionType> Action(
                           "Colored disassembly of strings of hex bytes")),
     cl::cat(MCCategory));
 
+static cl::opt<unsigned>
+    NumBenchmarkRuns("num-benchmark-runs",
+                     cl::desc("Number of runs for decoder benchmarking"),
+                     cl::cat(MCCategory));
```

Comment:
I don't think this really belongs to MCCategory

Reply:
I can remove it, just that most (except one) of the options here are in this category.